pkg/source/reconciler/mtsource/kafkasource.go

```diff
@@ -183,3 +188,46 @@ func (r *Reconciler) createCloudEventAttributes(src *v1beta1.KafkaSource) []duck
 	}
 	return ceAttributes
 }
+
+// initPolicyFromConfigMap reads predicates and priorities data from configMap
+func initPolicyFromConfigMap(ctx context.Context, configMapName string, policy *sts.SchedulerPolicy) error {
+	policyConfigMap, err := kubeclient.Get(ctx).CoreV1().ConfigMaps(system.Namespace()).Get(context.TODO(), configMapName, metav1.GetOptions{})
```

Comment:
`context.TODO()` -> `ctx`

Reply:
Sorry, didn't follow @lionelvillard. You are recommending setting the ctx via context.TODO() instead of being passed in as input?

Reply:
instead of `.Get(context.TODO(), ...`, what about just `.Get(ctx, ....`?

Reply:
Ohh yes, totally missed that. Good catch - thank you!

Reply:
Done